src/palladio/NodeParameter.h

```diff
@@ -144,6 +145,30 @@ const auto setStartRule = [](OP_Node* node, const std::wstring& s, fpreal t) {
 	node->setString(val, CH_STRING_LITERAL, START_RULE.getToken(), 0, t);
 };
 
+// -- SEED
+static PRM_Name SEED("randomSeed", "Seed");
+static PRM_Range SEED_RANGE(PRM_RANGE_RESTRICTED, std::numeric_limits<int>::lowest(), PRM_RANGE_RESTRICTED,
+                            std::numeric_limits<int>::max());
+static PRM_Name OVERRIDE_SEED("overrideRandomSeed", "Override");
+const std::string SEED_HELP = "Sets the random seed for all input primitives (attribute '" +
+                              PLD_RANDOM_SEED.toStdString() +
+                              "'). This value is ignored if the primitive attribute already exists and \"override\" is "
+                              "not checked. By default, the random seed is derived from the primitive position.";
+const std::string OVERRIDE_SEED_HELP =
+        "Force setting the random seed even if the corresponding primitive attribute already exists.";
+
+const auto getSeed = [](const OP_Node* node, fpreal t) -> int {
+	return node->evalInt(SEED.getToken(), 0, t);
+};
+
+const auto getOverrideSeed = [](const OP_Node* node, fpreal t) -> bool {
+	return static_cast<bool>(node->evalInt(OVERRIDE_SEED.getToken(), 0, t));
+};
```

Comment:
There's no actual need for lambdas here. Consider turning them into functions.

Reply:
replaced lambdas with real functions and separated body and declaration in cpp/h respectively.
Also sorted function bodys in cpp to match declaration order in h file.